### PR TITLE
Debug segfault reimplementation

### PR DESCRIPTION
**Pull request by KVS85**
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/6099/

### CODE REVIEW

File: js/client/modules/@arangodb/process-utils.js [hunk 1]
```diff
@@ -609,16 +609,8 @@ function executeAndWait (cmd, args, options, valgrindTest, rootDir, circumventCo
...
-    if (platform.substr(0, 3) !== 'win') {
-      // this shellscript will prevent cores from being writen on macos and linux.
-      args.unshift(cmd);
-      cmd = TOP_DIR + '/scripts/disable-cores.sh';
-    }
-  }
-
   if (options.extremeVerbosity) {
-    print(Date() + ' executeAndWait: cmd =', cmd, 'args =', args);
+    print(Date() + ' executeAndWait: cmd =', cmd, 'args =', args, 'circumventCores = ', circumventCores);
```
Comment (dothebart) [inline js/client/modules/@arangodb/process-utils.js]: please rename the `circumventCores` parameter `dummy` and remove it from here.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/6116/

File: js/client/modules/@arangodb/process-utils.js (hunk 1 above)
Comment (KVS85) [reply js/client/modules/@arangodb/process-utils.js]: Fixed.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/6121/

**Pull request merged**